tests/Person.spec.ts

```diff
@@ -7,6 +7,10 @@ import { PersonCreationViewPage } from "../pages/admin/PersonCreationView.page";
 import { PersonManagementViewPage } from "../pages/admin/PersonManagementView.page";
 import { HeaderPage } from "../pages/Header.page";
 import { faker } from "@faker-js/faker/locale/de";
+import { deletePersonen, getPersonId, createPersonWithUserContext } from "../base/api/testHelperPerson.page";
+import { getSPId } from "../base/api/testHelperServiceprovider.page";
+import { UserInfo } from "../base/api/testHelper.page";
```

Comment:
Muss das testHelperPerson heißen?

Reply:
Das war von mir so beabsichtigt. In den api/testHelper... Dateien sind nur Methoden; in der testHelper.page.ts habe ich die UserInfo "zentral" abgelegt. Die UserInfo könnte man auch in die testHelperPerson  packen, bin mir gerade aber nicht sicher.

Reply:
Ist ok, ergibt sinn, weil Userinfo und Personen... nicht zwingend zusammengehören